data/json/items/melee.json

```diff
@@ -94,7 +94,7 @@
     "price": 12500,
     "material": "wood",
     "techniques": [ "RAPID", "WBLOCK_2", "SWEEP" ],
-    "flags": [ "SHEATH_SWORD" ],
+    "flags": [ "SHEATH_SWORD", "FRAGILE_MELEE" ],
```

Comment:
This is still at least as solid as an ordinary stick, which doesnt get fragile melee, it's just too light and perhaps balanced wrong to optimum damage.

Reply:
If it was made out of balsa I could see it being fragile.